### DISCUSSION TITLE
code formatting

### DISCUSSION THREAD

**Discussion by cool-bigdogs-tshirt** [comfyanonymous/ComfyUI#587] [Ideas]
how willing would you be to go through the pain of getting code formatting set up? the longer this repo goes without it, the harder it will be to eventually add.

something like https://github.com/marketplace/actions/python-black-formatter-only-on-edited-files could help ease the codebase into it, but would also draw out the process.

**Comment by comfyanonymous** [comment]
Yeah I have been neglecting this a bit because python doesn't really suffer from formatting issues like C/C++ but I'll try to do better.
